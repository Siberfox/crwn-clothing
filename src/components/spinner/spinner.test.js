import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import Spinner from "./spinner.component";

it("should render Spinner component", () => {
  const wrapper = shallow(<Spinner />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
