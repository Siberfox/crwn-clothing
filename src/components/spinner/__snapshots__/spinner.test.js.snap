// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Spinner component 1`] = `
<styled.div>
  <styled.div />
</styled.div>
`;
